gc/base/standard/Scavenger.cpp

```diff
@@ -743,12 +745,14 @@ void
 MM_Scavenger::clearIncrementGCStats(MM_EnvironmentBase *env, bool firstIncrement)
 {
 	_extensions->incrementScavengerStats.clear(firstIncrement);
+	_cycleTimes.incrementEnd = 0;
```

Comment:
Can you make a comment that "Increment start time doesn't need to be cleared, it was set prior to calling this method"